shot/src/main/scala/com/karumi/shot/ShotPlugin.scala

```diff
@@ -72,4 +75,79 @@ class ShotPlugin extends Plugin[Project] {
     project.getExtensions.add(name, new ShotExtension())
   }
 
+  private def isLegacyGradleVersion(gradleVersion: String): Boolean = {
+    val versionNumbers = gradleVersion.split('.')
+    var major = 0
+    var minor = 0
+
+    if (versionNumbers.length > 0) {
+      major = versionNumbers(0).toInt
+    }
+    if (versionNumbers.length > 1) {
+      minor = versionNumbers(1).toInt
+    }
+
+    (major, minor) match {
+      case (major, _) if major > GRADLE_MIN_MAJOR => false
+      case (major, minor)
+          if major == GRADLE_MIN_MAJOR && minor >= GRADLE_MIN_MINOR =>
```

Comment:
How is it that we are using a `>` in the major version and `>=` for the minor version? Is this a bug? Can we safely change it to use the same semantics for both constants?

Reply:
I was talking with Sergio and it's not easy to give the same semantic. The best option could change the method's name but it's not good enough.

Reply:
It's ok in this case.